### PR TITLE
Add is_template to events table and add field to create form

### PR DESCRIPTION
**Pull request by abhishekbose87**
Fixes https://github.com/ReadyResponder/ReadyResponder/issues/254

### CODE REVIEW

**Comment by kgf** [comment]
Sadly, this also fails Travis. Are these failing for you locally ? `bundle exec rspec` 
To fix some of these, you can remove the associations between events and timecards. 
Any questions, just let me know !

Thanks for your help.
Kevin

**Comment by abhishekbose87** [comment]
Fixed the specs. Let me know if any other changes are required.

File: app/views/events/_form.html.erb [hunk 1]
```diff
@@ -6,6 +6,7 @@
     <%= f.input :description %>
     <%= f.input :status, :as => :select, :collection => Event::STATUS_CHOICES %>
     <%= f.input :category, :as => :select, :collection => Event::CATEGORY_CHOICES %>
+    <%= f.input :is_template, :as => :select, :collection => [true, false], :label => "Template" %>
```
Comment (kgf) [inline app/views/events/_form.html.erb:9]: Would this work better as a checkbox with a label ?

**Comment by kgf** [comment]
Consider my comment, but in any case, will need a rebase for the merge conflict.

Thanks for your help !
Kevin

**Comment by abhishekbose87** [comment]
The checkbox styling was not looking good out of the box, so I decided to go with the dropdown. Let me know if this works.
Also rebased my branch.

**Comment by kgf** [comment]
Thanks for trying the checkbox, and thanks for your help.

**Pull request merged**